CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -85,6 +86,7 @@ public NotesViewTopComponent() {
             }
             notesViewPane.updateNotesUI();
         });
+        jfxContainer.setMinimumSize(new Dimension(1000, 500));
```

Comment:
I'd prefer to make the view usable in the existing default width rather than simply changing the default width. Having tested this on my computer, the view by default now takes up more than half the width of my screen which just isn't practical (and reducing the width manually brings up the same issues as before). 

One idea might be to look what was done at the top of the Data Access View and have elements (e.g. buttons) on the screen flow onto the next line if they don't fit in the current width